src/photoswipe.d.ts

```diff
@@ -217,7 +217,39 @@ declare module 'photoswipe' {
     _onZoomPanUpdate(): void
   }
 
-  export default class PhotoSwipe extends PhotoSwipeBase {
+  export class PhotoSwipeLightbox extends PhotoSwipeBase {
+    constructor(options: PhotoSwipeOptions): PhotoSwipeLightbox
+
+    init(): void
+
+    onThumbnailsClick(e: MouseEvent): void
+
+    /**
+     * Get index of gallery item that was clicked.
+     */
+    getClickedIndex(e: MouseEvent): number
+
+    /**
+     * Load and open PhotoSwipe
```

Comment:
In some comments we have dots, in other not

Reply:
Comments copy-pasted from the original PhotoSwipe source. So, I think it's not an issue to leave it here for consistency.